modules/quarkus-native-binary-s2i-scripts/README.md

```diff
@@ -1,42 +1,94 @@
-# [Quarkus.io](http://quarkus.io) GraalVM Native S2I
+# [Quarkus.io](http://quarkus.io) GraalVM Native Binary S2i
 
-## OpenShift
+This s2i takes as input the executable image you want to deploy and build a minimal image to execute it. Unlike the `ubi-quarkus-native-s2i` it does not compile your application and build the native executable in OpenShift, rather expect the native executable as input.
 
-### Minishift 8 GB Set-Up recommendation
+## Instructions
 
-    minishift profile delete quarkus-s2i-native
-    minishift profile set quarkus-s2i-native
-    minishift config set memory 8192
-    minishift start
+The image is available on https://quay.io/repository/quarkus/ubi-quarkus-native-binary-s2i?tab=tags. 
+We recommend using the latest version.
 
-### OpenShift Build & Use
+## Building the image
 
-This S2I Builder image is available on https://quay.io/repository/quarkus/centos-quarkus-native-s2i.
+The first thing you need is to build the executable image for your application.
+Note that the image must be built for the _linux 64 bits_ architecture. 
 
-The quarkus.git/docs/src/main/asciidoc/openshift-s2i-guide.adoc documents how to use it,
-including how to increase the `BuildConfig`'s `limits`.
+For example, if you have a Maven project, use:
 
-Alternatively, you can locally build it inside your OpenShift cluster like this:
+```
+mvn clean install -Pnative -Dnative-image.docker-build=true
+```
 
-    oc new-build https://github.com/quarkusio/quarkus.git --context-dir=docker/centos-graal-maven-s2i --name quarkus-native-s2i
-    oc logs -f bc/quarkus-native-s2i
+## OpenShift Prerequisites
 
-To use the image locally built above instead of the one released to Quay.io, just use:
+You need to be connected to your OpenShift cluster with `oc`.
+Then, optionally, you can create a new project or check if the current project is appropriate.
 
-    oc new-app quarkus-native-s2i~https://github.com/quarkusio/quarkus-quickstarts --context-dir=getting-started --name=getting-started-native
+To create a new project, use:
 
-## Locally (only for testing)
+```
+oc new-project some-quarkus-test
+```
 
-### Local Build
+## Creating the build
 
-     cekit --verbose build --overrides-file centos-quarkus-native-s2i.yaml docker
+Once your OpenShift cluster is ready, create a new build as follows:
 
-### Local use
+```
+oc new-build --name=my-application \
+  --binary=true \
+  --docker-image=quay.io/quarkus/ubi-quarkus-native-binary-s2i:19.2.0
+```
 
-    sudo dnf install source-to-image
+Update the version (`19.2.0` in the snippet) to the latest version.
+You can also change the name of the build (`my-application`).
 
-    s2i build https://github.com/quarkusio/quarkus-quickstarts.git --context-dir=getting-started  quay.io/quarkus/centos-quarkus-native-s2i:latest getting-started-native
+## Triggering the build
 
-    docker run --rm -it -p 8080:8080 getting-started-native
+Trigger the image creation with:
+
+```
+oc start-build my-application --from-dir target
```

Comment:
I think it would be better to suggest that users use `--from-file` instead of `--from-dir target` since the later will upload everything in target dir.

Reply:
I suggest changing this to `oc start-build my-application --from-file target/*-runner`

Reply:
yes, but it breaks the .s2i/environment support. 
I may need to extend it to explain both.

Reply:
Fixed.